src/components/Page.js

```diff
@@ -0,0 +1,61 @@
+import React, { useState, useEffect } from 'react';
+import { useSelector, useDispatch } from 'react-redux';
+import { removeBook } from '../redux/books/books';
+
+function Page() {
```

Comment:
- [ ] Kindly try to follow the latest ES6 syntax guideline. Try to use the arrow function instead.